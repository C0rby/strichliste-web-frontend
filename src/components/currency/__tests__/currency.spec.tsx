import * as React from 'react';
import { cleanup, render } from 'react-testing-library';

import { IntlProvider } from 'react-intl';
import { Currency } from '../';

afterEach(cleanup);

describe('Currency', () => {
  it('matches the snapshot for english locales', () => {
    const { container } = render(
      <IntlProvider defaultLocale="en">
        <Currency value={120} />
      </IntlProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
