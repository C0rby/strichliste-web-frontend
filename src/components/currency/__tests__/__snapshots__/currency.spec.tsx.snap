// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Currency matches the snapshot for english locales 1`] = `
<span>
  €1.20
</span>
`;

exports[`Currency matches the snapshot for german locales 1`] = `
<span>
  1,20 €
</span>
`;
